### DISCUSSION TITLE
Draft idea for a Quick Zen like Little arc and a Zen Flow like arc Air Traffic Control

### DISCUSSION THREAD

**Discussion by 12th-devs** [zen-browser/desktop#2752] [General]
This is my draft for a Quick Zen and a Zen Flow that would function like Little Arc and Air Traffic Control respectively.

![Workspaces Sidebar=No, Pull Out Handle_=No](https://github.com/user-attachments/assets/80402cf4-9d27-4644-9837-53b988af452e)

![Workspaces Sidebar=No, Pull Out Handle_=Yes](https://github.com/user-attachments/assets/7570f025-59aa-41c1-8b11-521f10e75aa2)

![Workspaces Sidebar=Yes, Pull Out Handle_=No](https://github.com/user-attachments/assets/5996a277-d155-448b-b375-69da5ee30d83)

![Zen Flow](https://github.com/user-attachments/assets/1b91df93-eb6e-41df-b52c-701a07816301)
The / in the workspace would allow you to set the folder in the workspace it would appear in. if you have any feedback I'm all ears!

Note: I am not a Dev what so ever just a fan with ideas

**Comment by 12th-devs** [comment]
Hopefully we get something like this soon with folders coming soon

**Reply by vraravam** [reply]
Arc's ATC is cool, but imo, with Multi-Account Containers, one can achieve the same result, no?

**Reply by 12th-devs** [reply]
Yes, but not to the same degree of advancement. And no little arc alternative

**Comment by avelino** [comment]
I like the idea, I know Firefox has containers, but Air Traffic Control settings are much better (simpler), meaning we can use container structure and workspace in the background and create a "UI" to configure similar to what Arc has.

It would be important to give flexibility for users to contribute in a simple way, "removing regex" and putting intuitive forms, ex:
- contains text, "**buser**": matching "**buser**.com.br", "123.**buser**.com.br", "github.com/**buser**brasil" etc
